plugins/repository-s3/src/test/java/org/elasticsearch/cloud/aws/AwsS3ServiceImplTests.java

```diff
@@ -189,7 +191,8 @@ protected void launchAWSConfigurationTest(Settings settings,
                                               String expectedProxyPassword,
                                               String expectedSigner,
                                               Integer expectedMaxRetries,
-                                              boolean expectedUseThrottleRetries) {
+                                              boolean expectedUseThrottleRetries,
+                                              int expectedReadTimeout) {
```

Comment:
As per my previous comments, this has to be fixed if we don't define a default timeout.